packages/core/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@i18n-google-sheets/core",
```

```suggestion
  "name": "i18n-google-sheets",
```
Should be just name?

---

npm에 올라가는 패키지명을 이걸로 하면 어떨까요?